### ISSUE TITLE
Sort Order (asc/desc)

### ISSUE THREAD

**Issue by TekTimmy** [mikefarah/yq#1116]
I wish I could use yq to sort the YML data in ascending order.

If we have data1.yml like:
```yaml
countries:
  - Australia
  - Spain
  - Germany
```

And we run a command:

```bash
yq '.countries |= sort_by(.path, desc)' data1.yml
```

it could output

```yaml
countries:
  - Spain
  - Germany
  - Australia
```

**Comment by mikefarah** [comment]
Looking at what `jq` does, it has a `reverse` operator. I'll one one to yq too

**Comment by mikefarah** [comment]
Released in 4.21.1
